docs/switching_from/conda.md

```diff
@@ -0,0 +1,83 @@
+
+## Key Differences at a Glance
+
+| Task                        | Conda/Mamba                                       | Pixi                                                                 |
+|-----------------------------|---------------------------------------------------|----------------------------------------------------------------------|
+| Installation                | Requires an installer                             | Download and add to path (See [installation](../index.md))           |
+| Creating an Environment     | `conda create -n myenv -c conda-forge python=3.8` | `pixi init myenv` followed by `pixi add python=3.8`                  |
+| Activating an Environment   | `conda activate myenv`                            | `pixi shell` within the project directory                            |
+| Deactivating an Environment | `conda deactivate`                                | `exit` from the `pixi shell`                                         |
+| Running a Task              | `conda run -n myenv python my_program.py`         | `pixi run python my_program.py` (See [run](../reference/cli.md#run)) |
+| Installing a Package        | `conda install numpy`                             | `pixi add numpy`                                                     |
+| Uninstalling a Package      | `conda remove numpy`                              | `pixi remove numpy`                                                  |
+
+!!! warn "No `base` environment"
+    Conda has a base environment, which is the default environment when you start a new shell.
+    **Pixi does not have a base environment**. And requires you to install the tools you need in the project or globally.
+    Using `pixi global install bat` will install `bat` in a global environment, which is not the same as the `base` environment in conda.
+
+??? tip "Activating pixi environment in the current shell"
+    For some advanced use-cases, you can activate the environment in the current shell.
+    This uses the `pixi shell-hook` which prints the activation script, which can be used to activate the environment in the current shell without `pixi` itself.
+    ```shell
+    ~/myenv > eval "$(pixi shell-hook)"
+    ```
+
+## Environment vs Project
+`Conda` and `mamba` focus on managing environments, while `pixi` emphasizes projects.
+In `pixi`, a project is a folder containing a [manifest](../reference/project_configuration.md)(`pixi.toml`/`pyproject.toml`) file that describes the project, a `pixi.lock` lock-file that describes the exact dependencies, and a `.pixi` folder that contains the environment.
+
+This project-centric approach allows for easy sharing and collaboration, as the project folder contains all the necessary information to recreate the environment.
+It manages more than one environment for more than one platform in a single project, and allows for easy switching between them. (See [multiple environments](../features/multi_environment.md))
```

Comment:
Maybe a callout on how Pixi can lock for multiple platforms to help enable collaboration to highlight Pixi's additional abilities?